tests/ml_model/test_features.py

```diff
@@ -1,46 +1,64 @@
 import re
 from unittest import TestCase
 
+from credsweeper.app import APP_PATH
 from credsweeper.common.constants import Severity, MAX_LINE_LENGTH
 from credsweeper.credentials import Candidate, LineData
 from credsweeper.ml_model.features import SearchInAttribute, WordInPath, MorphemeDense, EntropyEvaluation, \
-    LengthOfAttribute
+    LengthOfAttribute, WordInPreamble, WordInTransition
 from credsweeper.ml_model.features.has_html_tag import HasHtmlTag
 from credsweeper.ml_model.features.is_secret_numeric import IsSecretNumeric
-from credsweeper.ml_model.features.word_in_line import WordInLine
+from credsweeper.ml_model.features.word_in_postamble import WordInPostamble
 from credsweeper.ml_model.features.word_in_value import WordInValue
+from credsweeper.utils import Util
 from tests import AZ_STRING
 
-RE_TEST_PATTERN = re.compile(r"(?P<variable>.*) (?P<separator>over) (?P<value>.+)")
+RE_TEST_PATTERN = re.compile(r"quick (?P<variable>brown fox) jumps (?P<separator>over) (?P<value>the lazy) dog")
 
 
 class TestFeatures(TestCase):
 
+    @staticmethod
+    def init_feature_search_comment(comment: str) -> SearchInAttribute:
+        feature = None
+        model_config = Util.json_load(APP_PATH / "ml_model" / "ml_config.json")
+        for fet in model_config["features"]:
+            if "SearchInAttribute" == fet["type"] and comment == fet.get("comment", ''):
+                assert feature is None, f"check duplication of '{comment}'"
+                feature = SearchInAttribute(**fet["kwargs"])
+        else:
+            assert feature is not None, f"missed SearchInAttribute for '{comment}'"
+        return feature
+
     def setUp(self):
+        self.maxDiff = None
+        self.model_config = Util.json_load(APP_PATH / "ml_model" / "ml_config.json")
         self.line_data = LineData(config=None,
                                   line=AZ_STRING,
                                   line_pos=0,
                                   line_num=1,
-                                  path="path.ext",
-                                  file_type="type",
+                                  path="./src/path.ext",
```

Comment:
```suggestion
                                  path="src/path.ext",
```
path looks like after glob